static/styles/zulip.css

```diff
@@ -40,6 +43,14 @@ body {
     font-size: 14px;
     line-height: calc(20 / 14);
     font-family: "Source Sans 3", sans-serif;
+
+    --color-message-time: hsla(0, 0%, 50%, 1);
+    --color-recipient-row-date: hsla(0, 0%, 37%, 1);
+}
+
+body.dark-theme {
+    --color-message-time: hsla(0, 0%, 50%, 1);
+    --color-recipient-row-date: hsla(0, 0%, 70%, 1);
```

Comment:
Based on discussion in https://chat.zulip.org/#narrow/stream/431-redesign-project/topic/color.20variables